machine-learning/tutorials/MovieRecommendation/Program.cs

```diff
@@ -83,12 +83,15 @@ public static ITransformer BuildAndTrainModel(MLContext mlcontext, IDataView tra
             };
 
             est = est.Append(mlcontext.Recommendation().Trainers.MatrixFactorization(options));
```

Comment:
Use full names for variables: est -> estimator.  Use a new variable as Append creates an EstimatorChain<TLastTransformer>.
var trainerEstimator = estimator.Append(mlContext.Recommendation().Trainers.MatrixFactorization(options));